features/FEATURE_EXPERIMENTAL_API/mbed_lib.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "experimental_api",
```

Comment:
Please add a "help" entry here, like in `rtos/source/TARGET_CORTEX/mbed_lib.json`, to describe what this feature is for and represents.

Reply:
This is still missing a `help` entry.

Reply:
It looks like we can't have `help` for features as a whole, I get this error when building an example:
`[ERROR] in ./mbed-os/features/FEATURE_EXPERIMENTAL_API/mbed_lib.json element : Additional properties are not allowed ('help' was unexpected)`

Reply:
Very sad :(((((((((((((

Reply:
OK